packages/lesswrong/server/resolvers/arbitalPageData.ts

```diff
@@ -3,19 +3,26 @@ import markdownIt from 'markdown-it'
 import markdownItMathjax from '../editor/markdown-mathjax'
 import { mjPagePromise } from '../editor/conversionUtils';
 import { trimLatexAndAddCSS } from '../editor/utils';
+import { ArbitalCaches } from '../../lib/collections/arbitalCache/collection';
+import { addCronJob } from '../cronUtil';
 
-const ArbitalPageData = `type ArbitalPageData {
+export const arbitalCacheExpirationMs = 15*60*1000;
```

Comment:
15 minutes seems a bit short, honestly, given that arbital is ~never(?) updated.  My guess is that this only really saves us CPU cycles in the case where a single page gets hit by a bot in a way that causes it to dodge the page cache?  Maybe it also helps if a new post links to arbital and gets popular.

But I think we can probably bump this to a few days, or something.